### PR TITLE
add a tz parameter to the date_format function

### PR DESCRIPTION
**Pull request by dvmlls**

### CODE REVIEW

File: R/trans-date.r [hunk 1]
```diff
@@ -68,7 +68,9 @@ date_breaks <- function(width = "1 month") {
 #' 
 #' @param format Date format using standard POSIX specification.  See
 #'  \code{\link{strptime}} for possible formats.
+#' @param tz a time zone name, see \code{\link{timezones}}. Defaults
+#'  to UTC
 #' @export
-date_format <- function(format = "%Y-%m-%d") {
-  function(x) format(x, format)
+date_format <- function(format = "%Y-%m-%d", tz='UTC') {
```
Comment (hadley) [inline R/trans-date.r]: Could you please add spaces around `=`?

**Comment by hadley** [comment]
Could you please also add a bullet to NEWS?

**Comment by hadley** [comment]
Perfect - thanks!

**Comment by dvmlls** [comment]
No no, thank you sir!

These packages are terrific.

**Pull request merged**